apps/condo/domains/ticket/utils/handlers.js

```diff
@@ -39,16 +63,35 @@ const detectEventTypes = ({ operation, existingItem, updatedItem }) => {
      * executor connected within create ticket operation or
      * executor connected/changed within update ticket operation
      */
-    result[EXECUTOR_CONNECTED_EVENT_TYPE] = isCreateOperation && !!nextExecutorId || isUpdateOperation && nextExecutorId && nextExecutorId !== prevExecutorId
+    result[EXECUTOR_CONNECTED_EVENT_TYPE] = isCreateOperation && !!nextExecutorId || isUpdateOperation && !!nextExecutorId && nextExecutorId !== prevExecutorId
```

Comment:
It can be simplified:

```suggestion
    result[EXECUTOR_CONNECTED_EVENT_TYPE] = !!nextExecutorId && (isCreateOperation || nextExecutorId !== prevExecutorId)
```

Reply:
https://github.com/open-condo-software/condo/blob/1ed20396933c30e561d58eb63f387e18eee51021/apps/condo/domains/ticket/utils/handlers.js

I've changed that code in my PR.